cranelift/filetests/filetests/isa/aarch64/traps.clif

```diff
@@ -17,6 +17,7 @@ block0(v0: i64, v1: i64):
 }
 
 ; block0:
+;   adds x3, x0, x1
```

Comment:
Wow, I'm surprised the test was missing this before -- that was a clear miscompilation that I guess we never saw only because we always used the result of `iadd_ifcout` directly as well. In any case, this PR fixes the issue!